helper/cli.c

```diff
@@ -255,25 +355,224 @@ static int cmd_call_odp_sys_info_print(struct cli_def *cli,
 	return CLI_OK;
 }
 
-static int cmd_call_odp_pktio_print(struct cli_def *cli,
-				    const char *command ODP_UNUSED,
-				    char *argv[], int argc)
+static odp_pktio_t get_pktio(struct cli_def *cli, char *argv[], int argc)
```

Comment:
I wouldn't make this a function. These parameters to get_pktio() look a little odd, when intuitively all it needs is a char *. But in that case you don't even need a function.

Some future command that needs a pktio might have some additional parameters, so check_num_args(..., 1) wouldn't work in that case.

Reducing code duplication is good, but in this case I would just repeat these few lines in each command.

Reply:
I have no strong opinions on this, though do slightly prefer the original way, so reverted to that in v4.